flyer_controller/HoverPoint hover_point
bool pause